File: Iniciante/1019/1019.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws IOException {
        Scanner i = new Scanner(System.in);
        int s = i.nextInt();
        int horas = s/3600;
        int minutos = (s%3600)/60;
        int segundos = (s%3600)%60;
        System.out.printf("%d:%d:%d\n", horas,minutos,segundos);
    }
}

File: Iniciante/1019/1019.cpp
#include <bits/stdc++.h>

using namespace std;

int main(){
    int s, horas, minutos, segundos;
    cin >> s;
    horas = s/3600;
    minutos = (s%3600)/60;
    segundos = (s%3600)%60;
    cout << horas << ":" << minutos << ":" << segundos << endl;
    return 0;
}